src/TelemetryConsumers/Orleans.TelemetryConsumers.Datadog/DatadogTelemetryConsumer.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Orleans.Runtime;
+using StatsdClient;
+
+namespace Orleans.TelemetryConsumers.Datadog
+{
+    public class DatadogTelemetryConsumer : IMetricTelemetryConsumer, IDisposable
+    {
+        private readonly DogStatsdService service;
+
+        public DatadogTelemetryConsumer()
+        {
+            var statsdConfig = new StatsdConfig();
```

Comment:
Does this not need customization by the user?

Reply:
@ReubenBond No - as it is now, it will send all (160+) metrics to datadog.

Reply:
Sample usage: https://github.com/PiotrJustyna/road-to-orleans/tree/main/3b

Reply:
Looking at the usage on the [Datadog site (under the .NET tab)](https://docs.datadoghq.com/developers/service_checks/dogstatsd_service_checks_submission/), they are configuring the statsd settings, so I think that we should also allow that here, as an option.